content/pt-br/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+título: Streaming de Eventos
+estado: Concluído
+categoria: conceito
+---
+
+## O que é isso
+
+O streaming de eventos é uma abordagem em que o software envia dados de eventos de um aplicativo para outro para comunicar continuamente o que eles estão fazendo.
+Imagine um serviço transmitindo tudo o que faz para todos os outros serviços.
+Cada atividade realizada por um serviço é referida como um evento, portanto, streaming de evento.
+Por exemplo, a NASDAQ obtém atualizações sobre preços de ações e commodities a cada segundo.
+Se você tivesse um aplicativo que monitorasse um conjunto específico de ações, gostaria de receber essas informações quase em tempo real.
+Yahoo! Finance fornece uma [API](/application-programming-interface/) que extrai da NASDAQ e envia (ou transmite) as informações (ou eventos) de seu aplicativo para qualquer aplicativo que o assine.
+Os dados enviados, bem como as alterações nesses dados (preços das ações), são os eventos, enquanto o processo de entregá-los a um aplicativo é o streaming de eventos.
+
+## Problema que aborda
+
+Tradicionalmente, o Yahoo! Finanças usaria solicitações TCP únicas.
```

Comment:
In this case I like "Yahoo! Finance" (line 14) instead of "Yahoo! Finanças".

Think about that "usaria requisições únicas de TCP."